blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+```
+pip install gqlalchemy
+```
+
+### Memgraph Lab installation
+Last piece of tech you will use is **[Memgraph Lab]()**. You will use it to connect to **Memgraph** and create visualizations. To check how to install **Memgraph Lab** and connect to **Memgraph**, you can go to one of the following pages for installation instructions for **Memgraph Lab as desktop application** for [Windows](https://memgraph.com/docs/memgraph-lab/installation/windows#step-2---installing-and-setting-up-memgraph-lab), [Mac](https://memgraph.com/docs/memgraph-lab/installation/macos#step-2---installing-and-setting-up-memgraph-lab) or [Linux](https://memgraph.com/docs/memgraph-lab/installation/linux#step-2---installing-and-setting-up-memgraph-lab). After you have installed **Memgraph Lab**, you can check instructions how to **[connect to Memgraph](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph)** with Memgraph Lab. 
+
+
+Now, when you have installed everything, we continue to the main topic of the tutorial: **importing large dataset** and **running graph analytics** with **Python**.
+
+## Connecting to Memgraph with GQLAlchemy
+
+Now position yourself in [jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics) you previously downloaded. We will start with the `qglalchemy` import and then we will connect to **Memgraph** using `host` and `port`. Afterward, we will clear our database, just in case.
+
+
+```python
+from gqlalchemy import Memgraph
+```
+
+
+```python
+memgraph = Memgraph("127.0.0.1", 7687)
+```
+
+
+```python
+memgraph.drop_database()
+```
+
+In the next few steps we will do the following:
+* we will show you how to import a large dataset from the **CSV** file inside **Memgraph** in terms of seconds. 
+* then you will see how to perform `PageRank` and `Louvain community detection` all by using **Python**
+
+## Importing data
+
+**CSV** files containing [**Facebook** dataset](https://snap.stanford.edu/data/gemsec-Facebook.html) have the following structure:
+```
+node_1,node_2
+0,1794
+0,3102
+0,16645
+```
+Dataset consists of Facebook pages (from November 2017). It represents a network of verified Facebook pages of different categories. Nodes represent the pages and relationships are mutual likes among them. The nodes are reindexed (starting from 0) to achieve a certain level of anonymity. Since **Memgraph** imports query faster when it has indexes, we will create one for all nodes with the label `Page` on the `id` property.  
+
+
+```python
+memgraph.execute(
+    """
+    CREATE INDEX ON :Page(id);
+    """
+)
+```
+
+Now we will make *full* usage of our "hack" from before. We will list through our local files in the `./data/facebook_clean_data/` folder and create the path from them concatenating our file names to the `/samples/` folder which will then represent the path to **CSV** files in *Docker* container.
```

Comment:
Now, to make full use of our "hack" from before, let's list through our local files...

..and create paths?  (not one path?)
from them or to them? 
not OUR file names, but THE files name

Now, to make full use of our "hack" from before, let's list through our local files in the `./data/facebook_clean_data/` folder to create their paths by concatenating the file names and the /samples/ folder. Those paths will then represent the paths to the.....